ext/libxml/config.w32

```diff
@@ -4,10 +4,10 @@ ARG_WITH("libxml", "LibXML support", "yes");
 
 if (PHP_LIBXML == "yes") {
 	if (CHECK_LIB("libxml2_a_dll.lib;libxml2_a.lib", "libxml") &&
-			CHECK_LIB("libiconv_a.lib;iconv_a.lib;libiconv.lib;iconv.lib", "libxml") &&
+			((PHP_ICONV == "no" || PHP_ICONV_SHARED) && CHECK_LIB("libiconv_a.lib;iconv_a.lib;libiconv.lib;iconv.lib", "libxml")
```

Comment:
I'm not sure I understand the new if condition, also because of the precedence of && and ||, please use parens to make it clear.

Reply:
Oh, that should have been
```suggestion
			(((PHP_ICONV == "no" || PHP_ICONV_SHARED) && CHECK_LIB("libiconv_a.lib;iconv_a.lib;libiconv.lib;iconv.lib", "libxml"))
```
Basically the same as https://github.com/php/php-src/pull/16798/files#r1842323752, but now it occurs to me that neither is right. because `PHP_ICONV` and `PHP_ZLIB` are not necessarily `"no"` or `"yes"` but could be an arbitrary string (a path). Thus in the second line we should have `PHP_ICONV != "no"` (instead of `PHP_ICONV == "yes"`), can then apply De Morgan's laws, and arrive at `A && B || !A`, i.e. `A → B`.